t04_refactored.py

```diff
@@ -583,7 +602,7 @@ def team_19_adv():
         print("You've chosen the special fruit, you've earned long-lasting prosperity!")
     else:
         print("Yummy! But not few and far between in these lands.")
-    kill_if_dead(dead)
+        kill_if_dead(dead)
```

Comment:
It looks like you edited another teams function... revert these!

Reply:
We didn't touch that, it became like that after a pull from master. Shall we still edit it?